x-pack/plugins/security_solution/public/common/components/visualization_actions/lens_embeddable.tsx

```diff
@@ -152,6 +153,7 @@ const LensEmbeddableComponent: React.FC<LensEmbeddableComponentProps> = ({
     inspectActionProps,
     timeRange: timerange,
     withActions,
+    lensMetadata: casesAttachmentMetadata,
```

Comment:
You might want to add it here as well: x-pack/plugins/security_solution/public/common/components/visualization_actions/actions.tsx